### PR TITLE
Infrastructure simplification

### PR DESCRIPTION
**Pull request by drwlrsn**
The goal is to get rid of the many complicated dependencies we have and instead just rely on Python and NPM.

### CODE REVIEW

**Comment by drwlrsn** [comment]
@rowan-maclachlan Thanks for the review. In the past we had a build and deploy process (and development really) that relied on Docker. Upon deep reflection and personal introspection it's become clear this rather bad solution. So much of this commit removes the files needed for Docker.

File: README.md [hunk 1]
```diff
@@ -3,28 +3,29 @@
...
-4. `$ docker-machine start && eval $(docker-machine env)`
-5. `$ cd .. && docker-compose -f docker-compose.yml -f docker-compose.dev.yml build`
-6. `$ docker-compose -f docker-compose.yml -f docker-compose.dev.yml run workstand python manage.py migrate`
-7. `$ docker-compose -f docker-compose.yml -f docker-compose.dev.yml run workstand python manage.py loaddata fixtures.yaml`
-8. `docker-compose -f docker-compose.yml -f docker-compose.dev.yml up`
-9. Visit http://192.168.99.100:8000/ (`docker-machine ls` to see IP address)
-10. Login with `u: admin@workstand.dev`and `p: bike!bike!`
-11. Stop docker any time with `docker-compose -f docker-compose.yml -f docker-compose.dev.yml down`
+2. `$ cd workstand`
+3. `$ brew install pyenv`
```
Comment (FeXd) [inline README.md]: Should we include a version number for brew installs?

Comment (FeXd) [reply README.md]: Latest:
`pyenv: 1.2.11`
`pipenv: 2018.11.26`

File: README.md [hunk 2]
```diff
@@ -3,28 +3,29 @@
...
+4. `$ brew install pipenv`
+5. `$ pyenv install 3.6.6`
```
Comment (FeXd) [inline README.md]: Getting an install error here: `zipimport.ZipImportError: can't decompress data; zlib not available`

Comment (FeXd) [reply README.md]: Seems to be fixed with this command:
`sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -target /`

Found via: https://stackoverflow.com/questions/52741673/how-can-i-install-zlib-on-mac-os-x-mojave-10-14

May want to make note about it?

File: README.md [hunk 3]
```diff
@@ -3,28 +3,29 @@
...
+6. `$ pipenv install --python 3.6.6 --dev`
+7. `$ pipenv shell`
+8. `$ cd bikeshop_project`
+9. `$ npm install`
+10. `$ ./manage.py migrate`
+11. `$ ./manage.py loaddata fixtures.yaml`
+12. `$ ./manage.py runserver`
+13. `$ npm start` _open a new terminal session to run this in workstand/bikeshop_project_
+14. Visit http://localhost:8000/
+15. Login with `u: admin@workstand.dev`and `p: bike!bike!`
```
Comment (FeXd) [inline README.md]: Success! Only hiccup was zlib.

File: README.md [hunk 4]
```diff
@@ -3,28 +3,29 @@
...
 
 ## Quick Start
 
-1. `$ cd workstand`
-2. `$ docker-machine start && eval $(docker-machine env)`
-3. `$ docker-compose -f docker-compose.yml -f docker-compose.dev.yml up`
-4. Stop docker at any time with `docker-compose -f docker-compose.yml -f docker-compose.dev.yml down`
+1. `$ cd workstand/bikeshop_project`
+2. `$ ./manage.py run server`
```
Comment (FeXd) [inline README.md]: This should be `$ ./manage.py runserver` (no space)

File: README.md [hunk 5]
```diff
@@ -3,28 +3,29 @@
...
 **Requirements**
 
-1. docker (ie: brew install docker)
-2. docker-compose (ie: brew install docker-compose)
-3. docker-machine (ie: brew install docker-machine)
-4. virtualbox (ie: brew install caskroom/cask/virtualbox)
-5. Node and NPM (installed in quick install)
+1. npm
+2. pyenv
+2. pipenv
```
Comment (FeXd) [inline README.md]: 3?

Also possibly add zlib, even tho likely works on all other versions of MacOS other than mojave out of the box.

**Comment by fspc** [comment]
I like it, good move! Docker should never be a dependency, that defeats the whole advantage of using it in the first place. This should make it easier now for me to release Docker development / demos for workstand, rather than doing this type of silliness: https://github.com/fspc/workstand

**Pull request merged**